src/pydrex/pathlines.py

```diff
@@ -1,109 +1,122 @@
 """> PyDRex: Functions for pathline construction."""
 import numpy as np
 from scipy import integrate as si
-from scipy import linalg as la
 
 from pydrex import logger as _log
+from pydrex import utils as _utils
 
 
 def get_pathline(
-    point, interp_velocity, interp_velocity_gradient, min_coords, max_coords
+    final_location,
+    get_velocity,
+    get_velocity_gradient,
+    min_coords,
+    max_coords,
+    max_strain,
+    **kwargs,
 ):
     """Determine the pathline for a particle in a steady state flow.
 
-    The pathline will intersect the given `point` and follow a curve determined by
-    the interpolated velocity gradient.
+    The pathline will terminate at the given `final_location` and follow a curve
+    determined by the velocity gradient. It works for both 2D (rectangular) and 3D
+    (orthopiped) domains, so long as the provided callables expect/return arrays of the
```

Comment:
Is orthopiped a word?

Reply:
I went a bit crazy with that, but someone else agrees with me: https://www.whatistoday.net/2020/04/cuboid-dilemma.html

Happy to change it to just 'box' or whatever is best.

Reply:
As you wish, but if you keep it the way it is, add the URL ;)